### ISSUE TITLE
pretrained model

### ISSUE THREAD

**Issue by indra4837** [muzi2045/second_TANET.pytorch#2]
Hey, great work for this repository. 

I am wondering if you can provide a pre-trained model for me to test the inference speed and accuracy? I am thinking of deploying this on Xavier and would like to know if its suitable before training on my custom dataset. 

It would be greatly appreciated, thank you.

**Comment by muzi2045** [comment]
you can train the model on the open source dataset like Nuscenes dataset and Lyft Dataset.

**Comment by indra4837** [comment]
Do you have pretrained model? Could you send it to me please at indra4837@gmail.com ? Thanks
